api/reference/authentications.adoc

```diff
@@ -0,0 +1,202 @@
+
+[[authentications]]
+== Authentications
+
+Management of Authentications is provided via the following collection:
+
+[source,data]
+----
+/api/authentications
+----
+
+Full CRUD actions on authentications is available:
+
+* link:#querying-authentications[Querying Authentications]
+* link:#creating-authentications[Creating Authentications]
+* link:#editing-authentications[Editing Authentications]
+* link:#deleting-authentications[Deleting Authentications]
+
+[[querying-authentications]]
+=== Querying Authentications
+
+Querying all authentications in the system is simply:
+
+----
+GET /api/authentications
+----
+
+Getting details on a specific Authentication:
+
+----
+GET /api/authentications/:id
+----
+
+One can also query authentications of a configuration_script_payload's manager as follows:
+
+----
+GET /api/configuration_script_payloads/:id/authentications
+----
+
+Or getting details on a specific Authentication:
+
+----
+GET /api/configuration_script_payloads/:id/authentications/:authentication_id
+----
+
+[[creating-authentications]]
+=== Creating Authentications
+
+Authentications can be created via a POST to the authentications collection or via the *create* action
+signature which also allows creation of multiple authentications in a single request.
+
+----
+POST /api/authentications
+----
+
+[source,json]
+----
+{
+  "description" : "Authentication Description",
+  "name" : "SomeCredentials",
+  "related" : {},
+  "type" : "ManageIQ::Providers::AnsibleTower::AutomationManager::Credential",
+  "manager_resource" : { "href" : "http://localhost:3000/api/providers/7" }
+}
+----
+
+or creating multiple authentications:
+
+[source,json]
+----
+{
+  "action" : "create",
+  "resources" : [
+    { "description" : "System Credentials", "name" : "SystemCreds", ... },
+    { "description" : "Admin Credentials",  "name" : "AdminCreds", ... },
+    ...
+  ]
+}
+----
+
+Optionally, Authentications can be created for a provider of a specific configuration script payload,
+thus eliminating the need of specifying a manager_resource.
```

Comment:
Similarly here with mixing case in referring to domain concepts - Authentications vs manager_resource